tests/Doctrine/Tests/DBAL/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -233,6 +233,29 @@ public function testListTableColumnsSameTableNamesInDifferentSchemas()
     }
 
     /**
+     * @group DBAL-1234
+     */
+    public function testListTableIndexesPrimaryKeyConstraintNameDiffersFromIndexName()
+    {
+        $table = new Table('list_table_indexes_pk_id_test');
+        $table->setSchemaConfig($this->_sm->createSchemaConfig());
+        $table->addColumn('id', 'integer', ['notnull' => true]);
```

Comment:
Please do not use short array syntax here, as we target this patch for `2.5` where we still support PHP `5.3`.